src/components/ViewAdmin.vue

```diff
@@ -0,0 +1,114 @@
+<!--
+SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<template>
+	<div id="context_chat">
+		<NcSettingsSection :name="t('context_chat', 'Status')">
+			<NcNoteCard v-if="stats.initial_indexing_complete" show-alert type="success">
+				{{ t('context_chat', 'The initial indexing run finished at: {date}.', {date: showDate(stats.intial_indexing_completed_at)}) }}
+			</NcNoteCard>
+			<NcNoteCard v-else type="warning">
+				{{ t('context_chat', 'The initial indexing is still running.') }}
+			</NcNoteCard>
+			<NcNoteCard v-if="stats.backend_available" show-alert type="success">
+				{{ t('context_chat', 'The Context Chat Backend app is installed and responsive.') }}
+			</NcNoteCard>
+			<NcNoteCard v-else type="warning">
+				{{ t('context_chat', 'The Context Chat Backend app is not installed or not responsing.') }}
+			</NcNoteCard>
+			<NcNoteCard v-if="stats.initial_indexing_complete && stats.eligible_files_count > stats.vectordb_document_counts['files__default'] * 1.2" type="warning">
+				{{ t('context_chat', 'Less files were indexed than expected. Only {percent}% files out of {eligibleCount} are in the VectorDB.', {percent: Math.round((stats.vectordb_document_counts['files__default'] / stats.eligible_files_count) * 100), eligibleCount: stats.eligible_files_count}) }}
+			</NcNoteCard>
+			<NcNoteCard type="info">
+				{{ t('context_chat', 'Eligible files for indexing: {count}', {count: stats.eligible_files_count}) }}
+			</NcNoteCard>
+			<NcNoteCard type="info">
+				{{ t('context_chat', 'Queued files for indexing: {count}', {count: stats.queued_files_count}) }}
+			</NcNoteCard>
+			<NcNoteCard v-for="(count, providerId) in stats.queued_documents_counts" :key="providerId" type="info">
+				{{ t('context_chat', 'Queued documents from provider {providerId} for indexing: {count}', {count, providerId}) }}
+			</NcNoteCard>
+			<template v-if="stats.vectordb_document_counts">
+				<NcNoteCard v-for="(count, providerId) in stats.vectordb_document_counts" :key="providerId" type="info">
+					{{ t('context_chat', 'Documents in VectorDB from provider {providerId}: {count}', {count, providerId}) }}
+				</NcNoteCard>
+			</template>
```

Comment:
a table inside the note card would look better here imo
in the form

| provider | queued documents | indexed documents |
|---|---|---|
|files__default|xx|xx|